sql/core/src/main/scala/org/apache/spark/sql/sources/filters.scala

```diff
@@ -38,6 +38,14 @@ case class EqualTo(attribute: String, value: Any) extends Filter
 
 /**
  * A filter that evaluates to `true` iff the attribute evaluates to a value
+ * null safe and equal to `value`.
```

Comment:
maybe this

```
Performs equality comparison, similar to [[EqualTo]]. However, this differs from [[EqualTo]] in that it returns true (rather than NULL) if both inputs are NULL, and false (rather than NULL) if one of the input is NULL and the other is not NULL.
```